reference-data-proxy/src/v1/controllers/party-db.controller.ts

```diff
@@ -11,7 +11,7 @@ export const lookup = async (req: Request, res: Response) => {
 
   const response = await axios({
     method: 'get',
-    url: `${partyDbURL}/${partyDbCompanyRegistrationNumber}`,
+    url: `${partyDbURL}/companyreg/${partyDbCompanyRegistrationNumber}`,
```

Comment:
as a sanity check, did the partyDbURL change? conscious that we didn't have `companyreg` before